data-structures/queues-1.js

```diff
@@ -0,0 +1,16 @@
+export function getFront(queue) {
+  return queue.peek() === undefined ? undefined : queue.peek();
```

Comment:
Or just `return queue.peek();` since it will return undefined if it's undefined.